NEWS

```diff
@@ -8,19 +8,36 @@
   Major changes between OpenSSL 1.1.0i and OpenSSL 1.1.1 [in pre-release]
 
       o Support for TLSv1.3 added
```

Comment:
Only one line for the largest new feature? How modest!

Reply:
One could perhaps talk about the state machine rewrite, extension handling rewrite, and packet-construction code that were added to support it.

Reply:
The state machine rewrite and packet construction code were done for 1.1.0. There were of course new states added for TLSv1.3. The packet *writing* code is new in 1.1.1

Reply:
Of course I realise what I just wrote about packets made absolutely no sense... :-)

Reply:
@mattcaswell, how about stealing some lines from your [blog post](https://www.openssl.org/blog/blog/2018/02/08/tlsv1.3) or mentioning [RFC8446](https://tools.ietf.org/html/rfc8446) :wink: ?

Reply:
See #7167